### PR TITLE
Modify css to avoid text overlap in french language

### PR DESCRIPTION
**Pull request by christophe-rouille**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Adding flex on blocks overlaping image
| Type?             | bug fix 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes PrestaShop/PrestaShop#26103
| How to test?      | Go to BO > Modules > Module Manager<br>Click on Configure button of Google Analytics<br>See the text "La performance de votre merchandising" no overlap the image below<br>Before<br> image<br>After<br>image
| Possible impacts? | display module in BO

### CODE REVIEW

**Comment by christophe-rouille** [comment]
Style on #google_analytics_content img has been added

File: views/css/ganalytics.css [hunk 1]
```diff
@@ -27,9 +27,15 @@
...
 }
 
 #advantages_list div > img {
 	margin: 1em;
 	width: 60px;
 }
+
+#google_analytics_content img {
+    max-width: 100%;
+}
```
Comment (NeOMakinG) [inline views/css/ganalytics.css:41]:
```suggestion
}

```

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by hibatallahAouadni** [comment]
Hello @christophe-rouille 

LGTM, QA :heavy_check_mark: 

https://www.awesomescreenshot.com/video/5942730?key=3df20974a9e62c5d32c1d74dd113213c

Thanks!

**Comment by NeOMakinG** [comment]
Thanks @christophe-rouille

**Pull request merged**